src/Domain/DomainHelpers.cpp

```diff
@@ -693,9 +690,26 @@ sph_wedge_coordinate_maps(
   const auto compression = domain::CoordinateMaps::EquatorialCompression{
       aspect_ratio, index_polar_axis};
 
+  // Set up rotation map:
+  const auto rotation =
+      which_wedges == ShellWedges::AllAndHalvesXY
+          ? domain::CoordinateMaps::DiscreteRotation<3>{OrientationMap<3>(
+                std::array<Direction<3>, 3>{{Direction<3>::upper_eta(),
```

Comment:
for some reason I still need the `std::array`s - I'm assuming it's because of the `auto` but if you prefer I remove them and use the explicit type (I kept it as `auto` because I've used `auto` above for the other maps) I can change it, but I left it as is for now

Reply:
Deleting only the `std::array<Direction<3>, 3>` should work I think (don't delete the double braces). Don't worry about this though.